do-app-platform-op-cli/google-workspace/workspace-settings.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
🤔 Should this be `workspace-settings.json.example`?

Reply:
🤔 In all the other platform examples, we have defined this file as workspace-settings.json, and tried to clearly instruct admins to edit this file (without the need to rename it) before adding it as a secret.

Reply:
Fair enough. Consistency!